app/docker/__module.js

```diff
@@ -2,11 +2,12 @@ import angular from 'angular';
 
 import containersModule from './containers';
 import { componentsModule } from './components';
+import { networksModule } from './views/networks';
```

Comment:
IMO move networksModule to `./networks`

Reply:
I agree about putting the migrated network related components in a feature folder: /app/docker/networks
Can I shift the new components there now?

![image](https://user-images.githubusercontent.com/83188384/160594153-2efc22f5-b9ca-4ac7-a57a-d80446c2abd4.png)

Reply:
[app/docker/__module.js](https://github.com/portainer/portainer/blob/5b13de3865d869eaa9f9b433472f0c48a5d16574/app/docker/__module.js#L5)

Reply:
updated with the above file structure as I'm migrating